Traceback (most recent call last):
  File "main.py", line 37, in <module>
    main()
  File "main.py", line 28, in main
    test_kernel(17, 16, 16)
  File "main.py", line 21, in test_kernel
    test.test_all()
  File "/home/bcheng/workspace/dev/pycuda/cuda/ex6/testbench.py", line 107, in test_all
    instance.run()
  File "/home/bcheng/workspace/dev/pycuda/cuda/ex6/matmul.py", line 86, in run
    self._dot[self.BPG, self.TPB](dA, dB, dC)
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/compiler.py", line 798, in __call__
    kernel = self.specialize(*args)
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/compiler.py", line 809, in specialize
    kernel = self.compile(argtypes)
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/compiler.py", line 828, in compile
    kernel.bind()
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/compiler.py", line 542, in bind
    self._func.get()
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/compiler.py", line 420, in get
    ptx = self.ptx.get()
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/compiler.py", line 391, in get
    **self._extra_options)
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/cudadrv/nvvm.py", line 496, in llvm_to_ptx
    ptx = cu.compile(**opts)
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/cudadrv/nvvm.py", line 233, in compile
    self._try_error(err, 'Failed to compile\n')
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/cudadrv/nvvm.py", line 251, in _try_error
    self.driver.check_error(err, "%s\n%s" % (msg, self.get_log()))
  File "/home/bcheng/workspace/tools/anaconda3/envs/pycuda/lib/python3.6/site-packages/numba/cuda/cudadrv/nvvm.py", line 141, in check_error
    raise exc
numba.cuda.cudadrv.error.NvvmError: Failed to compile

<unnamed> (70, 31): parse expected comma after getelementptr's type
NVVM_ERROR_COMPILATION
